jax/_src/util.py

```diff
@@ -493,6 +493,16 @@ def maybe_named_axis(axis, if_pos, if_named):
     named = True
   return if_named(axis) if named else if_pos(pos)
 
+def get_logger(name: Optional[str] = None, level: int = logging.INFO) -> logging.Logger:
+  logger = logging.getLogger(name=name)
+  # TODO: Enable file handler attachments and level setting from config
+  stream = logging.StreamHandler()
```

Comment:
I think you should not add a handler. The behavior you're going to get if you don't do this is that the messages will be propagated to the root logger. That's the behavior we want! In particular, we want to interoperate with absl logging, and absl logging installs an alternative handler on the root logger. We do not want to force things to go to stderr for example if the root logger isn't configured that way.

Just remove the handler code.